src/Sylius/Bundle/CoreBundle/Context/SessionAndChannelBasedCartContext.php

```diff
@@ -75,17 +68,17 @@ public function getCart(): OrderInterface
             throw new CartNotFoundException(null, $exception);
         }
 
-        if (!$this->session->has(sprintf('%s.%s', $this->sessionKeyName, $channel->getCode()))) {
+        if (!$this->cartSessionStorage->hasCartId($channel->getCode())) {
```

Comment:
What would you say about passing whole object and naming of method `hasCartIdForChannel`?